src/source/index.ts

```diff
@@ -1,8 +1,26 @@
+import { source_simple_chip } from "./source_simple_chip"
+import { z } from "zod"
+
 export * from "./source_simple_capacitor"
 export * from "./source_simple_resistor"
 export * from "./source_simple_diode"
 export * from "./source_simple_ground"
-export * from "./source_simple_bug"
+
+/**
+ * @deprecated Use source_simple_chip instead. This will be removed in a future version.
+ */
+export const old_source_simple_bug = z.object({
```

Comment:
We should keep the name the same, we want to make ot so anyone importing from @tscircuit/soup or @tscircuit/props doesnt have their code broken